packages/trader/src/AppV2/Components/PayoutInfoModal/payout-info-modal.tsx

```diff
@@ -0,0 +1,29 @@
+import { CaptionText, Modal } from '@deriv-com/quill-ui';
+import { Localize } from '@deriv/translations';
+import React from 'react';
+
+const PayoutInfoModal = ({ body_content }: { body_content: React.ReactNode }) => {
```

Comment:
Sorry for another silly question, but where are we using this component? Also, there is a  simple button tag inside without any `className`. Should it be replaced with `<Button />` from quill?

Reply:
We're not using this component because design team changed it during implementation phase. This, however, might be useful in the future